src/idpyoidc/server/oidc/token.py

```diff
@@ -26,14 +26,21 @@ class Token(token.Token):
     name = "token"
     default_capabilities = None
 
+    helper_by_grant_type = {
```

Comment:
are these the default that can be overloaded from the general config?

Reply:
No this is a specification of what helpers there are.
Which you eventually use if specified in grant_types_supported in the configuration.